modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/GridCacheDatabaseSharedManager.java

```diff
@@ -412,8 +413,16 @@ private void registerSystemView() {
                 try {
                     List<MetastorageView> data = new ArrayList<>();
 
-                    metaStorage.iterate("", (key, val) ->
-                        data.add(new MetastorageView(key, IgniteUtils.toStringSafe(val))), true);
+                    metaStorage.iterate("", (key, valBytes) -> {
+                        try {
+                            Serializable val = metaStorage.marshaller().unmarshal((byte[])valBytes, U.gridClassLoader());
+
+                            data.add(new MetastorageView(key, IgniteUtils.toStringSafe(val)));
+                        }
+                        catch (IgniteCheckedException ignored) {
+                            data.add(new MetastorageView(key, null));
```

Comment:
I suggest to add to javadoc info about that `null` value is can be raw data.
As alternative way show something like `[Raw data]` instead of `null`.